mender-update/update_module/v3/platform/boost_filesystem/fs_operations.cpp

```diff
@@ -128,6 +128,13 @@ error::Error UpdateModule::PrepareFileTree(
 
 	const fs::path file_tree_path {path};
 
+	boost::system::error_code ec;
+	fs::remove_all(file_tree_path, ec);
```

Comment:
There was some reason why I avoided using the versions that take the `error_code` and relied on exceptions instead. But I don't recall what it was, something mentioned in the documentation. Please double-check it.

Reply:
Hmm, I can't see anything obvious. You mean [this documentation](https://www.boost.org/doc/libs/1_74_0/libs/filesystem/doc/reference.html#remove_all), right? I checked the "Error reporting" section as well.

Reply:
Hmm, don't see anything obvious either, but I do recall reading those docs and going with exceptions after that. :sweat_smile: We'll see. :)